### DISCUSSION TITLE
Hyperlinks

### DISCUSSION THREAD

**Discussion by Derivedbetter** [RafaelGB/obsidian-db-folder#751] [Q&A]
I am trying to create a CRM database with people listed including their social media profiles etc. It seems that I can't get hyperlinks to show properly. Is there some setting that I'm missing, or is this just not possible?

![image](https://user-images.githubusercontent.com/93354976/211603473-b57af8df-2393-48e6-9c55-1eeca3f9755a.png)

**Comment by RafaelGB** [comment]
Hello @Derivedbetter ,

https://user-images.githubusercontent.com/11924043/211610299-dcda52e2-c420-4efb-b15f-2e036a5d1d19.mov

You have activated the embed media.

Some links like images could be displayed as embed, but most of them not (future feature?)
